plugins/SSH.koplugin/main.lua

```diff
@@ -13,7 +13,7 @@ local _ = require("gettext")
 
 --Only tested on Kindle, should also work on SDL(linux)
 --Kobo and other devices need to be tested first.
-if not Device:isSDL() and not Device:isKindle() then return { disabled = true, } end
+if not Device:isSDL() and not Device:isKindle() and not Device:isKobo() then return { disabled = true, } end
```

Comment:
Hm, we might want to distinguish more between SDL platforms. Dropbear makes no sense on Ubuntu Touch, Sailfish or in the AppImage, does it? But we can easily do that during the build. Then here you could do something like `if isSDL() then if not lfs.attributes(dropbear) return disabled`.

Just brainstorming. :-)

Reply:
yes, that's a better way to do it, just test the presence of dropbear at runtime.